README.md

```diff
@@ -2,17 +2,19 @@
 
 Nango requires specific components and configurations to operate correctly within a Kubernetes cluster. Key dependencies include:
 
-- **[Temporal](https://temporal.io/)**: Nango relies on Temporal for syncs and actions.
-The environment variables `TEMPORAL_ADDRESS` and `TEMPORAL_NAMESPACE` must be set which
-you can receive from a Nango developer. Additionally, `TEMPORAL_NAMESPACE.key`
-and `TEMPORAL_NAMESPACE.crt` files need to be configured as Kubernetes secrets.
+- **[Elasticsearch](https://www.elastic.co/)**: Nango relies on Elasticsearch for logging.
+Nango can operate without running Elasticsearch and the environment variable `NANGO_LOGS_ENABLED`
+determines if logs are ingested and displayed in the Nango UI. If you are running Ela
+need to set:
```

Comment:
```
If you are running Ela
need to set:
```
something is missing. right?

Reply:
Yeah, got cut off somehow. Updated with https://github.com/NangoHQ/nango-helm-charts/pull/2/commits/d67e23d7bacdba718a3f15911b2f4d4172409408